app/spec_helpers/event_emitter.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Events = require('events');
+
+class EventEmitter extends Events {}
+
+const meh = new EventEmitter;
+meh.__id__ = Math.random();
```

Comment:
This file should have been deleted. The event emitter didn't work... I used `dnode` instead. The gulp task and electron are in different processes therefore this isn't a singleton.